tests/functional/ddb/test_put.py

```diff
@@ -42,12 +42,6 @@ def tearDown(self):
         yaml.YAML(typ='safe').constructor.yaml_constructors.update(
             self.original_tag_handlers)
 
-    def assert_yaml_response_equal(self, response, expected):
```

Comment:
I'll check with @stealthycoin on this, but agree it's strange. If we aren't needing it then should remove the constructors too.

Reply:
My guess is this was copy pasted from another test class and not removed. Lets just remove it.

Reply:
Removed all yaml references.

Reply:
I'm not sure if we should be removing this. The ruamel.yaml has a fun property that the `yaml_constructors` property is a class property of the underlying constructor class so if you mutate the `constructor` for any instances of `YAML`, it will affect all instances of `YAML` and this is most notable across tests as we use the same process. This is noted in the commit that added it: https://github.com/aws/aws-cli/commit/823c480d3245a0ffe6c48d8acec4bd46bd8bb426. We also do this else where to preserve state from test to test with our yaml loader: https://github.com/aws/aws-cli/blob/v2/tests/unit/customizations/cloudformation/__init__.py.

Reply:
Thanks. Should we take the opportunity and do as the linked one suggests and reuse the `BaseYAMLTest` defined there to make this more clear and standardized?

https://github.com/aws/aws-cli/blob/v2/tests/unit/customizations/cloudformation/__init__.py#L22

Reply:
I'd say eventually. Honestly, I rather turn this all into a pytest fixture. I'd say hold off on it for now. I'd really like us just to get this upgrade merged.